ddp-workspace/projects/ddp-pancan/src/app/components/header/header.component.html

```diff
@@ -1,10 +1,27 @@
 <header class="header" [ngClass]="{'panel-opened': isPanelOpened, 'header_shadow': isPageScrolled}">
     <div class="header-content">
         <a class="project-logo" routerLink="/">
-            <img class="project-logo__img" src="assets/images/logo.svg" [alt]="'App.Header.Logo.ImageAlt' | translate">
-            <p class="project-logo__text" [innerHTML]="'App.Header.Logo.Title' | translate"></p>
+            <img class="project-logo__img"
+                 width="55px"
```

Comment:
Shouldn't we change the logo size for different screens like it is implemented in other studies?

Reply:
good notice! fixed